sdk/couchbase-core/src/mgmtx/mgmt.rs

```diff
@@ -86,7 +86,7 @@ impl<C: Client> Management<C> {
         self.http_client
             .execute(req)
             .await
-            .map_err(|e| error::Error::new_message_error("could not execute request").with(e))
+            .map_err(|e| error::Error::new_message_error(format!("could not execute request: {e}")))
```

Comment:
I can see the logic of moving the implementation details out of an error struct, and replacing that with logging, but I don't quite see the point of moving it from a structured form to a merged error message string?